cmd/tsconnect/wasm/wasm_js.go

```diff
@@ -81,6 +81,14 @@ func newIPN(jsConfig js.Value) map[string]any {
 		authKey = jsAuthKey.String()
 	}
 
+	jsHostname := jsConfig.Get("hostname")
+	var hostname string
+	if jsHostname.Type() == js.TypeString {
```

Comment:
Is `jsHostname` (or any of the other params) re-used? If not, wouldn't it be better for GC to declare it in the if scope like `if jsHostname := jsConfig.Get("hostname"); jsHostname == js.TypeString {`?

Reply:
Good point. I was being consistent with the other option fields (`jsAuthKey`, etc.) but they can all be rewritten in this style.